.github/workflows/e2e-tests.yml

```
@@ -55,8 +55,8 @@ jobs:
         cp .env.example .env
         echo "FALKORDB_HOST=localhost" >> .env
         echo "FALKORDB_PORT=6379" >> .env
-        echo "FLASK_SECRET_KEY=test-secret-key-for-ci" >> .env
-        echo "FLASK_DEBUG=False" >> .env
+  echo "FASTAPI_SECRET_KEY=test-secret-key-for-ci" >> .env
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 58: could not find expected ':'

(syntax-check)